src/gdcdictionary/schemas/exposure.yaml

```diff
@@ -641,6 +644,13 @@ properties:
       - Unknown
       - Not Reported
       - Not Allowed To Collect
+      - "1"
+      - "2"
+      - "3"
+      - "4"
+      - "5"
+      - "6"
+      - "7"
```

Comment:
for my knowledge: as part of the sorting rules, are we moving deprecated enums to the end?